jaeger-core/src/main/java/io/jaegertracing/internal/TracingFactory.java

```diff
@@ -0,0 +1,77 @@
+package io.jaegertracing.internal;
+
+import io.jaegertracing.internal.clock.Clock;
+import io.jaegertracing.internal.metrics.Metrics;
+import io.jaegertracing.spi.BaggageRestrictionManager;
+import io.jaegertracing.spi.Reporter;
+import io.jaegertracing.spi.Sampler;
+import io.opentracing.ScopeManager;
+
+import java.util.List;
+import java.util.Map;
+
+public class TracingFactory {
```

Comment:
add javadoc

Reply:
Ya. Just need to stabilize the API, then will do.